StreamDeckPlugin/Actions/PageRightAction.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Threading.Tasks;
+using SharpDeck;
+using SharpDeck.Events.Received;
+using SharpDeck.Manifest;
+
+namespace ArkhamOverlaySdPlugin.Actions {
+    [StreamDeckAction("Page Right", "arkhamoverlay.pageright")]
+    public class PageRightAction : StreamDeckAction {
+        protected async override Task OnKeyDown(ActionEventArgs<KeyPayload> args) {
+            foreach (var cardButtonAction in CardButtonAction.ListOf) {
+                if (!cardButtonAction.IsVisible) {
+                    continue;
+                }
+
+                cardButtonAction.Page++;
```

Comment:
they can go to the right indefinitely, and it would be hard to stop them because I don't actually know how many cards there are- but if they go up and back down, it resets to the starting page, so I don't think it's a big deal.